volttrontesting/testutils/test_fixtures.py

```diff
@@ -1,12 +1,14 @@
 import pytest
-from volttrontesting.fixtures.volttron_platform_fixtures import volttron_instance_web
```

Comment:
lets rename this file to be test_docker_wrapper instead of test_fixtures  or put the docker_wrapper tests in a new file.  I don't see all the stuff in the diff here.

Reply:
Will do. I'll also remove the test_web_setup_properly since that doesn't test the docker_wrapper. Why was it there in the first place?